appengine/multitenancy/README.md

```diff
@@ -1,56 +1,13 @@
-## Multitenancy Using Namespaces Sample
+## Google App Engine Namespaces
 
-This is a sample app for Google App Engine that exercises the [namespace manager Python API](https://cloud.google.com/appengine/docs/python/multitenancy/multitenancy).
+This sample demonstrates how to use Google App Engine's [Namespace Manager API](https://cloud.google.com/appengine/docs/python/multitenancy/multitenancy) in Python.
 
-See our other [Google Cloud Platform github
-repos](https://github.com/GoogleCloudPlatform) for sample applications and
-scaffolding for other python frameworks and use cases.
+### Running the sample
 
-## Run Locally
-1. Install the [Google Cloud SDK](https://cloud.google.com/sdk/), including the [gcloud tool](https://cloud.google.com/sdk/gcloud/), and [gcloud app component](https://cloud.google.com/sdk/gcloud-app).
-2. Setup the gcloud tool.
+You can then run the sample on your development server:
+        
+      $ dev_appserver.py .
```

Comment:
We have to be consistent with docs:
https://cloud.google.com/appengine/docs/python/tools/uploadinganapp

gcloud is still in preview.